Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine as builder
+FROM quay.io/openshift-pipeline/golang:alpine as builder
```

Comment:
do we automatically mirror this image?

Reply:
Yep, but see https://github.com/openshift-pipelines/vote-api/pull/6 😉